### ISSUE TITLE
ShowTitleBar=false but keep window controls

### ISSUE THREAD

**Issue by milos12345** [sourcechord/FluentWPF#46]
If I hide the Title Bar then the window controls disappear. Is there a way to remove it and keep these controls (so the content, such as tabs, could be moved to the top of the window, as in Edge)

**Comment by sourcechord** [comment]
Hi @milos12345.

I released FluentWPF 0.7.0.
There is new property for customize AcrylicWindow's title bar.
If you set `AcrylicWindow.TitleBar` attached property, you can define AcrylicWindows's titlebar content.
```xml
<fw:AcrylicWindow x:Class="FluentWPFSample.Views.AcrylicWindow4"
                  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
                  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
                  xmlns:fw="clr-namespace:SourceChord.FluentWPF;assembly=FluentWPF"
                  xmlns:local="clr-namespace:FluentWPFSample.Views"
                  mc:Ignorable="d"
                  Title="AcrylicWindow4" Height="450" Width="800"
                  AcrylicWindowStyle="NoIcon">
    <fw:AcrylicWindow.TitleBar>
        <Grid WindowChrome.IsHitTestVisibleInChrome="True">
            <Grid.ColumnDefinitions>
                <ColumnDefinition />
                <ColumnDefinition Width="Auto"/>
            </Grid.ColumnDefinitions>
            <Menu Style="{StaticResource AcrylicMenuStyle}"
                  HorizontalAlignment="Left" VerticalAlignment="Center">
                <MenuItem Header="MenuItem1">
                    <MenuItem Header="MenuItem3" />
                    <MenuItem Header="MenuItem4" />
                    <MenuItem Header="Sub">
                        <MenuItem Header="MenuItem3" />
                        <MenuItem Header="MenuItem4" />
                    </MenuItem>
                </MenuItem>
                <MenuItem Header="MenuItem2">
                    <MenuItem Header="MenuItem3" />
                    <MenuItem Header="MenuItem4" />
                    <MenuItem Header="Sub">
                        <MenuItem Header="MenuItem3" />
                        <MenuItem Header="MenuItem4" />
                    </MenuItem>
                </MenuItem>
            </Menu>

            <TextBox Grid.Column="1"
                     Margin="2" Width="200"
                     Height="24" FontSize="12" Padding="2" BorderThickness="1"
                     Style="{StaticResource TextBoxRevealStyle}"
                     fw:Extensions.PlaceholderText="Placeholder"/>
        </Grid>
    </fw:AcrylicWindow.TitleBar>
    <Grid>
    </Grid>
</fw:AcrylicWindow>
```
![image](https://user-images.githubusercontent.com/14825436/58892337-f9267700-8728-11e9-9a1f-ef14a93f9bea.png)

Please try it.

**Comment by sourcechord** [comment]
I'll close this issue.
If you have any problems, please reopen this issue.
